### ISSUE TITLE
Change integration testing

### ISSUE THREAD

**Issue by chrislovecnm** [cockroachdb/cockroach-operator#310]
Currently, our e2e testing runs the operator locally.  In order to run integration testing, we need to launch the operator inside of a container, rather than run the binary locally.  Use bazel k8s rules to launch the container.

**Comment by chrislovecnm** [comment]
@keith-mcclellan I don't think we actually need this for the release.  As part of the GKE testing, I am installing the operator in the default namespace. We should improve that, but otherwise, e2e running locally is ok for now.

**Comment by keith-mcclellan** [comment]
If we can cover this in integration testing instead I'm okay closing this out.

**Comment by chrislovecnm** [comment]
I think we should do some of this, so I am just removing the next release label
